hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -322,6 +326,16 @@ public Response put(
     }
   }
 
+  private static boolean hasTrailingSlash(String keyPath) {
+    return keyPath.charAt(keyPath.length() - 1) == '/';
+  }
```

Comment:
Looks like this method is not used anymore, I think we can remove it.

Reply:
Done.